### ISSUE TITLE
Update old packages

### ISSUE THREAD

**Issue by bjones1** [PreTeXtBook/pretext-cli#496]
Running `poetry update` followed by `poetry show --outdated` produces the following.

```
pdfcropmargins 1.0.9   2.0.3  A command-line program to crop the margins of PDF files, with many options.
pydantic       1.10.12 2.1.1  Data validation and settings management using python type hints
pyee           9.0.4   11.0.0 A port of node.js's EventEmitter to python.
pypdf2         2.5.0   3.0.1  A pure-python PDF library capable of splitting, merging, cropping, and transforming PDF files
watchdog       2.3.1   3.0.0  Filesystem events monitoring
```

From our `pyproject.toml`, we have:

```
pdfCropMargins = "~1.0.9"
PyPDF2 = "~2.5"
watchdog = "^2"
```

Can we update all these to the latest versions? Note that:
- `pydantic-xml` requireds `pydantic` 1.0.
- `pyee` is a version-locked `playwright` dependency.

**Comment by oscarlevin** [comment]
We likely can and should upgrade these.  Note that the pypdf2 and pdfcropmargins were set that way because there was an incompatibility race between them, so it took some testing to find a combination that worked.  Hopefully in the new versions this is gone.

**Comment by bjones1** [comment]
Thanks! What tests can I run to check that these are behaving properly?

**Comment by bjones1** [comment]
Per the [docs](https://github.com/abarker/pdfCropMargins#whats-new), pdfCropMargins no longer uses PyPDF2. So, I'm removing the PyPDF2 dependency and updating the pdfCropMargins version. It looks like this is used for LaTeX image generation, so I'll run the tests which (I think) check this.

**Comment by oscarlevin** [comment]
That's great.  I just checked the support thread and found the issue, which showed up when latex-images were generated.  So if those work correctly, we should be all set.

**Comment by oscarlevin** [comment]
Closed by #499
